lcc_comchain_base/models/res_partner.py

```diff
@@ -163,6 +163,7 @@ def comchainCreateOrder(self, comchain_address, amount):
             # TODO: case with contact of a company ?
             order_vals = {
                 "partner_id": partner.id,
+                "user_id": None,
```

Comment:
It's not a good id to set user_id to none, because some ir.rules depends on its and some user will see all orders because of none in user_id:

Example: 
![image](https://github.com/Lokavaluto/lokavaluto-addons/assets/995787/6fad68ec-ea2c-4aa2-87e7-24f6b41c2562)

Reply:
@njeudy maybe we can force the user_id Odoo_bot?

Reply:
ok for odoobot

Reply:
solved :-)